src/pages/AddCar.jsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addCar } from '../redux/xers/carXer';
+import '../assets/App.css';
+import '../assets/addcar.css';
+import { useEffect } from 'react';
+
+function AddCar() {
```

Comment:
- [ ]  Please, kindly refactor this functional component  to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.

Refactoring is easy:

```javascript
const AddCar= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.